android/src/toga_android/fonts.py

```diff
@@ -28,69 +28,69 @@ class Font:
     def __init__(self, interface):
         self.interface = interface
 
-    def typeface(self, *, default=Typeface.DEFAULT):
-        cache_key = (self.interface, default)
-        if typeface := _FONT_CACHE.get(cache_key):
-            return typeface
-
-        font_key = self.interface._registered_font_key(
-            self.interface.family,
-            weight=self.interface.weight,
-            style=self.interface.style,
-            variant=self.interface.variant,
-        )
+        # 1. Check for a cached typeface.
         try:
-            font_path = _REGISTERED_FONT_CACHE[font_key]
+            self.native_typeface = _FONT_CACHE[self.interface]
         except KeyError:
-            # Not a pre-registered font
-            if self.interface.family not in SYSTEM_DEFAULT_FONTS:
-                print(
-                    f"Unknown font '{self.interface}'; "
-                    "using system font as a fallback"
+
+            # 2. Check for a system font.
+            try:
+                self.native_typeface = {
+                    # SYSTEM is left as a placeholder.
```

Comment:
`None` would be a more appropriate placeholder; that way, the type of `native_typeface` would be `Typeface | None`, rather than `Typeface | str` with only one string value actualy used.